oxford-metadata.ttl

```diff
@@ -148,28 +148,33 @@
     a :Category ;
     rdfs:label "Physical properties" .
 
-:CellMembraneCurrent
+:IonicCurrent
+    a rdfs:Class ;
+    rdfs:label "Ionic currents" ;
+    rdfs:comment "All terms that represent a current anywhere in the cell." .
+
+:CellMembraneCurrentRelated
     a :Category ;
-    rdfs:label "Membrane currents" ;
+    rdfs:label "Membrane currents and their properties" ;
     rdfs:comment "Annotations associated with currents across the cell membrane." .
 
 :StimulusCurrent
-    a :CellMembraneCurrent ;
+    a :CellMembraneCurrentRelated ;
     a :ChasteMetadata ;
-    rdfs:label "Stimulus current properties" .
+    rdfs:label "Stimulus current and its properties" .
 
-:MembranePump
-    a :CellMembraneCurrent ;
+:MembranePumpRelated
+    a :CellMembraneCurrentRelated ;
     rdfs:label "Pumps" ;
-    rdfs:comment "Proteins that use energy in ATP to pump ions against their electrochemical gradients." .
+    rdfs:comment "Terms related to proteins that use energy in ATP to pump ions against their electrochemical gradients." .
 
 :MembraneExchanger
```

Comment:
This class has not been renamed; similarly for others. We should probably be consistent!

Reply:
like now?